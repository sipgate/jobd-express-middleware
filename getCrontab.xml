<?xml version="1.0"?>
<methodResponse>
	<params>
		<param>
			<value>
				<struct>
					<member>
						<name>serial</name>
						<value>
							<i4>1550489257</i4>
						</value>
					</member>
					<member>
						<name>faultString</name>
						<value>
							<string>ok</string>
						</value>
					</member>
					<member>
						<name>systemName</name>
						<value>
							<string>portd</string>
						</value>
					</member>
					<member>
						<name>faultCode</name>
						<value>
							<i4>200</i4>
						</value>
					</member>
					<member>
						<name>jobs</name>
						<value>
							<struct>
								<member>
									<name>resendFailedFaxes</name>
									<value>
										<struct>
											<member>
												<name>interval</name>
												<value>
													<struct>
														<member>
															<name>timeout</name>
															<value>
																<i4>300</i4>
															</value>
														</member>
														<member>
															<name>daysOfMonth</name>
															<value>
																<string>*</string>
															</value>
														</member>
														<member>
															<name>minutesOfHour</name>
															<value>
																<string>2</string>
															</value>
														</member>
														<member>
															<name>hoursOfDay</name>
															<value>
																<string>6,8,10,12,14,16</string>
															</value>
														</member>
													</struct>
												</value>
											</member>
											<member>
												<name>maxFailuresAllowedBeforeNotification</name>
												<value>
													<i4>3</i4>
												</value>
											</member>
										</struct>
									</value>
								</member>
							</struct>
						</value>
					</member>
				</struct>
			</value>
		</param>
	</params>
</methodResponse>
