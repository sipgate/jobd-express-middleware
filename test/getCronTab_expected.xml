<methodResponse>
	<params>
		<param>
			<value>
				<struct>
					<member>
						<name>faultString</name>
						<value>
							<string>ok</string>
						</value>
					</member>
					<member>
						<name>faultCode</name>
						<value>
							<i4>200</i4>
						</value>
					</member>
					<member>
						<name>systemName</name>
						<value>
							<string>my-little-system</string>
						</value>
					</member>
					<member>
						<name>jobs</name>
						<value>
							<struct>
								<member>
									<name>my-job</name>
									<value>
										<struct>
											<member>
												<name>interval</name>
												<value>
													<struct>
														<member>
															<name>daysOfMonth</name>
															<value>
																<string>*</string>
															</value>
														</member>
														<member>
															<name>daysOfWeek</name>
															<value>
																<string>*</string>
															</value>
														</member>

														<member>
															<name>hoursOfDay</name>
															<value>
																<string>*</string>
															</value>
														</member>
																<member>
															<name>minutesOfHour</name>
															<value>
																<string>*</string>
															</value>
														</member>
														<member>
															<name>secondsOfMinute</name>
															<value>
																<string>*</string>
															</value>
														</member>
														<member>
															<name>timeout</name>
															<value>
																<i4>300</i4>
															</value>
														</member>
													</struct>
												</value>
											</member>
											<member>
												<name>maxFailuresAllowedBeforeNotification</name>
												<value>
													<i4>3</i4>
												</value>
											</member>
										</struct>
									</value>
								</member>
							</struct>
						</value>
					</member>
				</struct>
			</value>
		</param>
	</params>
</methodResponse>